cloud/pkg/cloudhub/level.go

```diff
@@ -0,0 +1,108 @@
+package cloudhub
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"time"
+
+	"gopkg.in/yaml.v2"
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/beehive/pkg/core"
+	beehiveContext "github.com/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/beehive/pkg/core/model"
+)
+
+type CloudConfig struct {
+	Cloud struct {
+		IP          string `yaml:"ip"`
+		SignalRange int    `yaml:"signal_range"`
+	} `yaml:"cloud"`
+}
+
+type LevelMessage struct {
+	IP    string `json:"ip"`
+	Level int    `json:"level"`
+}
+
+// read config
+func loadCloudConfig() CloudConfig {
+	data, err := ioutil.ReadFile("./config.yaml")
+	if err != nil {
+		klog.Fatalf("Failed to read config file: %v", err)
+	}
+
+	var config CloudConfig
+	err = yaml.Unmarshal(data, &config)
+	if err != nil {
+		klog.Fatalf("Failed to unmarshal config file: %v", err)
+	}
+
+	return config
+}
+
+type CloudHubModule struct{}
+
+// Name
+func (ch *CloudHubModule) Name() string {
+	return "cloudhub"
+}
+
+// Group
+func (ch *CloudHubModule) Group() string {
+	return "hub"
+}
+
+// Enable
+func (ch *CloudHubModule) Enable() bool {
+	return true
+}
```

Comment:
It would be better to predefine it as a constant and then return it, you can refer to https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/cloudhub/cloudhub.go#L82